config/initializers/prometheus.rb

```diff
@@ -1,2 +1,5 @@
 require 'prometheus/active_job_subscriber'
 require 'prometheus/active_record'
+
+Yabeda.configure! if Yabeda.respond_to?(:configure!)
+Prometheus::ActiveJobSubscriber.attach_to :active_job
```

Comment:
This needs to be called after the `Yabeda.configure!` otherwise it will instanciate a `Prometheus::ActiveJobSubscriber` instance in the class definition which is not possible because it uses a not yet defined `Yabeda.que` method